client/asset/btc/btc.go

```diff
@@ -856,10 +859,15 @@ type ExchangeWalletFullNode struct {
 	*intermediaryWallet
 }
 
+// ExchangeWalletAuthenticator implements the asset.Authenticator.
+type ExchangeWalletAuthenticator struct {
+	*ExchangeWalletFullNode
+}
+
```

Comment:
No need for a new type

Reply:
Thanks for the review @JoeGruffins, the reason for the new type was because `zec` had the `NoAuth` field set to `true` and letting `baseWallet` keep the exported methods means it will be called for `zec`.

Reply:
Ugh, I missed the zec issue.  That's really unfortunate fallout.

Reply:
Even with the new `asset.Authenticator` we still need to be able to tell the frontend that the wallet deos not require password(esp. for BTC clones like `zec` since seeded and token wallets ain't an issue). We can either do it with `noauth` or  `traits`.